packages/react-components/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-components
+
+**React Components components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```

Comment:
```suggestion
**Converged Fluent UI React components**
```
(this should probably also link to a doc/demo site for converged components once we have one)